src/main/resources/application.yml

```diff
@@ -6,3 +6,7 @@ chs:
   kafka:
     api:
       endpoint: ${CHS_KAFKA_API_URL:localhost}
+
+springfox:
+  documentation:
+    auto-startup: false
```

Comment:
not required if you update private sdk to latest version and remove Werror setting from men